patientIdentifierType.md

```diff
@@ -0,0 +1,104 @@
+# PatientIdentifierType
+
+## Overview
+Administrators define what types of identifiers they will collect. These range from National ID numbers, to driver's license numbers, to per-hospital medical record numbers.
+
+### Properties on PatientIdentifierType:
+
+* format: a regular expression defining what the identifier text should contain
+
+* formatDescription: An optional description of the regular expression (used to explain the requirements of the regular expression in terms a user would understand). For example, a regular expression like `\d{4,8}` could have a description like "Must be a number between 4 and 8 digits in length."
+
+* required: a true/false whether every patient MUST have this type
+
+* checkDigit: a true/false whether this identifier has a checkdigit at the end
+
+* validator: full class name of an [IdentifierValidator](https://docs.openmrs.org/doc/org/openmrs/patient/IdentifierValidator.html) (e.g., [`org.openmrs.patient.impl.LuhnIdentifierValidator`](https://docs.openmrs.org/doc/org/openmrs/patient/impl/LuhnIdentifierValidator.html))
+
+* locationBehavior: "REQUIRED" if a location must be associated with the identifier; "NOT_USED" if the identifier does require a location
+
+## Available operations
+
+1. [List PatientIdentifierType resources](#List-patientidentifierType-resource)
+2. [Create a patientIdentifierType record](#create-a-patientidentifierType)
+3. [Update a patientIdentifierType record](#update-a-patientIdentifierType)
+4. [Delete a patientIdentifierType record](#delete-a-patientIdentifierType)
+
+### List PatientIdentifierType resource
+
+* #### List patientIdentifierType
+
+    Fetch all non-retired patientIdentifierTypes resources that match any specified parameters otherwise fetch all non-retired patients. Returns a `200 OK` status with the patientIdentifierType response. If the user is not logged in it returns `401 Unauthorized` status is returned.
+
+    ```console
+    GET /patientidentifiertype?
+        &q=**searchqQuery**
```

Comment:
`?q={searchQuery}` doesn't make sense here, since you are describing getting all patient identifier types. If searching for patient identifier types was supported (I don't think it is), you would want to describe it separately.

Reply:
yes, to list the patientIdentiferTypes nothing is required, will update